src/main/resources/sql/migrations/tenant/V32__insert_errorcode.sql

```diff
@@ -0,0 +1,32 @@
+INSERT INTO `errorcode` (`TRANSACTION_TYPE`, `ERROR_MESSAGE`, `ERROR_CODE`, `RECOVERABLE`)
+VALUES (
+    'COLLECTION-MPESA',
+    'The transaction is being processed',
+    '500.001.1001',
+    true
+);
+
+INSERT INTO `errorcode` (`TRANSACTION_TYPE`, `ERROR_MESSAGE`, `ERROR_CODE`, `RECOVERABLE`)
+VALUES (
+    'COLLECTION-MPESA',
+    'DS timeout.',
+    '1037',
```

Comment:
Isnt ds timeout and 1037 the same thing?

Can you also include errors for error.contains network and connection - the 2 most common errors?

Reply:
no they are not same. One is `transactionMessage` and another one is `errorCode`.

Reply:
didn't get the second question can you elaborate on it.